src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -46,42 +49,59 @@ public async Task<IActionResult> Search(StandardViewModel model)
                 return RedirectToAction(nameof(Index), new { model.Id });
             }
 
-            var standards = await _apiClient.GetStandards();
+            var standards = await _standardVersionApiClient.GetAllStandardVersions();
 
-            model.Results = standards.Where(s => s.Title.Contains(model.StandardToFind, StringComparison.InvariantCultureIgnoreCase)).ToList();
+            model.Results = standards.Where(s => s.Title.Contains(model.StandardToFind, StringComparison.InvariantCultureIgnoreCase))
+                .GroupBy(
+                    s => s.IFateReferenceNumber,
+                    (key, stds)  => new StandardVersion()
+                    {
+                        IFateReferenceNumber = key,
+                        Title = stds.FirstOrDefault()?.Title
+                    })
+                .ToList();
 
             return View("~/Views/Application/Standard/FindStandardResults.cshtml", model);
         }
 
         [HttpGet("standard/{id}/confirm-standard/{standardCode}")]
         [ApplicationAuthorize(routeId: "Id")]
-        public async Task<IActionResult> ConfirmStandard(Guid id, int standardCode)
+        public async Task<IActionResult> ConfirmStandard(Guid id, string standardCode)
         {
             var application = await _apiClient.GetApplication(id);
-            var standardViewModel = new StandardViewModel { Id = id, StandardCode = standardCode };
+            var standardViewModel = new StandardVersionViewModel { Id = id, StandardCode = standardCode };
             if (!CanUpdateApplicationAsync(application))
             {
                 return RedirectToAction("Applications", "Application");
             }
 
-            var standards = await _apiClient.GetStandards();
-            standardViewModel.SelectedStandard = standards.FirstOrDefault(s => s.StandardId == standardCode);
-            standardViewModel.ApplicationStatus = await ApplicationStandardStatus(application, standardCode);
+            var standards = (await _standardVersionApiClient.GetAllStandardVersions())
```

Comment:
I'd also be tempted here to add a GetStandardVersionsByIFateReferenceNumber, We already have a similiar but by LarsCode so would simplify the call and logic, thoughts?

Reply:
See previous note.

Reply:
Added Sub task to SV-564 https://skillsfundingagency.atlassian.net/browse/SV-822